login_canary/locale/pl/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,73 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-08-17 21:18+0200\n"
+"PO-Revision-Date: 2015-08-17 21:18+0200\n"
+"Last-Translator: Janusz Harkot <janusz.harkot@gmail.com>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
+"|| n%100>=20) ? 1 : 2);\n"
+
+#: apps.py:10
+msgid "Login Canary"
+msgstr "Login Canary"
+
+#: templates/login_canary/logged_in_body.txt:1
+#, python-format
+msgid ""
+"Dear %(user)s,\n"
+"\n"
+"On %(datetime)s somebody successfully logged in to your account on "
+"%(service_name)s.\n"
+"\n"
+"User agent: %(user_agent)s\n"
+"IP-Address: %(remote_addr)s\n"
+"\n"
+"Kind regards."
+msgstr ""
+"Witaj %(user)s,\n"
+"\n"
+"Dnia %(datetime)s ktoś z powodzeniem zalogował się na Twoje konto w "
+"serwisie %(service_name)s.\n"
+"\n"
+"Przeglądarka: %(user_agent)s\n"
+"Adres IP: %(remote_addr)s\n"
+"\n"
+"Z poważaniem."
+
+#: templates/login_canary/logged_in_subject.txt:1
+#, python-format
+msgid "Successful login on %(service_name)s"
+msgstr "Poprawne logowanie w %(service_name)s"
+
+#: templates/login_canary/login_failed_body.txt:1
+#, python-format
+msgid ""
+"Dear %(user)s,\n"
+"\n"
+"On %(datetime)s somebody tried to login to your account on %(service_name)s "
+"but provided invalid credentials.\n"
+"\n"
+"Kind regards."
+msgstr ""
+"Witaj %(user)s,\n"
+"\n"
+"Dnia %(datetime)s ktoś próbował zalogować się na Twoje konto w serwisie %(service_name)s "
+"jednakże uzył niepoprawnych danych logowania.\n"
```

Comment:
I would replace `jednakże uzył niepoprawnych danych logowania` with `jednak zostały użyte niepoprawne dane logowania`. It fixes spelling mistake (`uzyl` instead of `użył`) and makes it gender neutral :)